themes/bootstrap3/templates/search/facet-list.phtml

```diff
@@ -28,36 +28,38 @@
     <label><?=$this->translate('Sort') ?></label>
     <div class="btn-group">
       <?php foreach ($this->sortOptions as $key => $sort): ?>
-        <a href="<?=$urlBase . '&amp;facetpage=1&amp;facetsort=' . urlencode($key) ?>" class="btn btn-default js-facet-sort<?php if ($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>" data-lightbox-ignore><?=$this->translate($sort) ?></a>
+        <a href="<?=$urlBase . '&amp;facetpage=1&amp;facetsort=' . urlencode($key) . '&amp;contains=' . urlencode($contains) ?>" class="btn btn-default js-facet-sort<?php if($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>"><?=$this->translate($sort) ?></a>
       <?php endforeach; ?>
     </div>
   </div>
 <?php endif; ?>
-<div class="lightbox-scroll full-facets">
+
+<form>
```

Comment:
Do you think it might be worth putting the filter input next to the sort options rather than below them? It looks vertically crowded and there seems to be plenty of horizontal space:

![image](https://github.com/vufind-org/vufind/assets/309069/49f47bc7-e6fc-471c-8821-884122d5df53)

I also wonder if we should introduce a new language string for the placeholder here, in case somebody wants to translate it independently of the existing "Filter" string. We could use the language tools to copy the existing string to a new key if you think that's worth doing.

Reply:
I moved the filter into a separate btn-group with its own laben next to the Sort controls.
Not sure whether creating an independent translation key would cause any benefit right now, but if you think so please also let me know what a good label / content would be.